tickit/devices/eiger/eiger_settings.py

```diff
@@ -0,0 +1,154 @@
+    countrate_correction_count_cutoff: int = field(default=1000, metadata=rw_int())
+    data_collection_date: str = field(
+        default="2021-30-09T16:30:00.000-01:00", metadata=ro_str()
+    )
+    description: str = field(
+        default="Simulated Eiger X 16M Detector", metadata=ro_str()
+    )
+    detector_distance: float = field(default=2.0, metadata=rw_float())
+    detector_number: str = field(default="EIGERSIM001", metadata=ro_str())
+    detector_readout_time: float = field(default=0.01, metadata=rw_float())
+    _element: str = field(
+        default="Co", metadata=rw_str(allowed_values=[e.name for e in KA_Energy])
+    )
+    flatfield: List[List[float]] = field(
+        default_factory=lambda: [[]],
+        metadata=field_config(value_type=AccessMode.FLOAT_GRID),
+    )
+    flatfield_correction_applied: bool = field(default=True, metadata=rw_bool())
+    frame_time: float = field(default=0.12, metadata=rw_float())
+    kappa_increment: float = field(default=0.0, metadata=rw_float())
+    kappa_start: float = field(default=0.0, metadata=rw_float())
+    nimages: int = field(default=1, metadata=rw_int())
+    ntrigger: int = field(default=1, metadata=rw_int())
+    number_of_excuded_pixels: int = field(default=0, metadata=rw_int())
+    omega_increment: float = field(default=0.0, metadata=rw_float())
+    omega_start: float = field(default=0.0, metadata=rw_float())
+    phi_increment: float = field(default=0.0, metadata=rw_float())
+    phi_start: float = field(default=0.0, metadata=rw_float())
+    photon_energy: float = field(default=6930.32, metadata=rw_float())
+    pixel_mask: List[List[int]] = field(
+        default_factory=lambda: [[]],
+        metadata=field_config(value_type=AccessMode.UINT_GRID),
+    )
+    pixel_mask_applied: bool = field(default=False, metadata=rw_bool())
+    roi_mode: str = field(
+        default="disabled", metadata=rw_str(allowed_values=["disabled", "4M"])
+    )
+    sensor_material: str = field(default="Silicon", metadata=ro_str())
+    sensor_thickness: float = field(default=0.01, metadata=ro_float())
+    software_version: str = field(default="0.1.0", metadata=ro_str())
+    threshold_energy: float = field(default=4020.5, metadata=rw_float())
+    trigger_mode: str = field(
+        default="exts", metadata=rw_str(allowed_values=["exts", "ints", "exte", "inte"])
+    )
+    two_theta_increment: float = field(default=0.0, metadata=rw_float())
+    two_theta_start: float = field(default=0.0, metadata=rw_float())
+    wavelength: float = field(default=1e-9, metadata=rw_float())
+    x_pixel_size: float = field(default=0.01, metadata=ro_float())
+    x_pixels_in_detector: int = field(default=FRAME_WIDTH, metadata=rw_int())
+    y_pixel_size: float = field(default=0.01, metadata=ro_float())
+    y_pixels_in_detector: int = field(default=FRAME_HEIGHT, metadata=rw_int())
+
+    # TODO: Make a decorator so this hacky fix isn't needed
```

Comment:
See https://github.com/dls-controls/tickit/pull/25/files#r742072717

Reply:
"Resolve or remove this" - me

Reply:
This doesn't exist anymore either